core/src/core-api/_internalLinkManager.js

```diff
@@ -87,13 +88,23 @@ export class linkManager extends LuigiCoreAPIBase {
   fromClosestContext() {
     this.options.fromContext = null;
     this.options.fromClosestContext = true;
+    this.options.fromParent = false;
     return this;
   }
 
   fromVirtualTreeRoot() {
     this.options.fromContext = null;
     this.options.fromClosestContext = false;
     this.options.fromVirtualTreeRoot = true;
+    this.options.fromParent = false;
+    return this;
+  }
+
+  fromParent() {
```

Comment:
Why the need to add fromParent or just to have it in sync with the rest 🤔

Reply:
Yes, to keep it in sync.
Tested different scenarios with our current functionality like:
_...fromClosestContext().getCurrentRoute()_
_...fromContext('ctx').getCurrentRoute()_
and realized that getParent().getCurrentRoute is not supported.